app/Composers/App.php

```diff
@@ -4,15 +4,15 @@
 
 use Roots\Acorn\View\Composer;
 
-class Alert extends Composer
+class App extends Composer
 {
     /**
      * List of views served by this composer.
      *
      * @var array
      */
     protected static $views = [
-        'components.alert'
+        '*',
```

Comment:
Maybe instead of doing this, we could rename to Composers\Layouts\App.php and remove $views all together. Then it will automatically be included with layouts/app.blade.php.

Reply:
I considered that, but resorted to this for simplicities sake as to not complicate anything with more directories, etc. – I figured we would express the additional ways Composers can be used/auto-detected within' the docs.